framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -204,7 +204,8 @@ def hide_launch_and_setup_pse_button(self):
 
     @property
     def include_branch_button(self):
-        if self.is_launch_fragment and self.is_simulation_copy and not self.is_pse_launch:
+        is_branch = common.get_from_session(common.KEY_BRANCH)
```

Comment:
Let's not link `SimulatorFragmentRenderingRules` object with the session, send the `is_branch` flag as a parameter to keep consistency. We might refactor this part soon, so it will be easier to keep it separate.